SoraUnitySdkSamples/Assets/SoraSample.cs

```diff
@@ -298,10 +301,23 @@ void Update()
             }
         }
     }
+    void OnChangeRoute()
+    {
+        if (audioOutputHelper == null)
+        {
+            return;
+        }
+        Debug.LogFormat("OnChangeRoute : " + (audioOutputHelper.IsHandsfree() ? "ハンズフリー OFF" : "ハンズフリー ON"));

```

Comment:
すいません、元ネタは私なのですが、ログとしては逆です。（ボタンは自分のステータスと逆の表示をするものなので大丈夫です）
```suggestion
        Debug.LogFormat("OnChangeRoute : " + (audioOutputHelper.IsHandsfree() ? "ハンズフリー ON" : "ハンズフリー OFF"));

```

Reply:
失礼しました。修正します。